### ISSUE TITLE
dataset understanding question

### ISSUE THREAD

**Issue by Duperr** [zuluokonkwo/Encrypted-Network-Traffic-Classification-with-Higher-Order-Graph-Neural-Network#9]
I have a question. 

- From my understanding of your article, the dataset of GCN is to compose each flow into a graph, and then perform operations such as model training. 

- If the number of flows is extremely large when the dataset is relatively large, will the computational efficiency of constructing each flow as a graph as the dataset of GCN affect the computational efficiency?

**Comment by zuluokonkwo** [comment]
Yes, computational efficiency can be affected if flows are really large,  but I still need to understand your definition of "large".

If by "large" you mean the number of packets in a flow then yes this might slow things down, because your graphs will be very dense and hard to train.

if by "large" you mean number of flow samples, then this is relative to how large the individual graphs are. If the individual graphs are not dense, then the number of samples won't really affect the training process in an adverse way (its a good thing to have a lot of training samples).
